### PR TITLE
Add reflective alpine images to other components

### PR DESCRIPTION
**Pull request by seemethere**
Was running into issues where the alpine images were failing on armhf / s390x machines

### CODE REVIEW

File: rpm/Makefile [hunk 1]
```diff
@@ -1,11 +1,12 @@
 ARCH=$(shell uname -m)
+ALPINE:=$(shell $(CURDIR)/../detect_alpine_image)
```
Comment (andrewhsu) [inline rpm/Makefile]: would prefer this var be called `ALPINE_IMG`

Comment (andrewhsu) [inline rpm/Makefile]: is `detect_alpine_image` part of this PR?

Comment (seemethere) [inline rpm/Makefile]: No It was included as part of: https://github.com/docker/docker-ce-packaging/pull/4

Comment (seemethere) [inline rpm/Makefile]: Change to ALPINE_IMG as of c423146

**Comment by andrewhsu** [comment]
LGTM

**Pull request merged**